snuba/datasets/errors_replacer.py

```diff
@@ -277,20 +304,20 @@ def process_message(self, message: ReplacementMessage) -> Optional[Replacement]:
         ):
             return None
         elif type_ == "end_delete_groups":
-            processed = process_delete_groups(event, self.__required_columns)
+            processed = process_delete_groups(message, self.__required_columns)
```

Comment:
why did this have to change? `event` used to be `message.data` now it's just `message`?

Reply:
`message` here is a `ReplacementMessage` object which just has that event data and the replacement type. Before it was just passing the event data to the processors but now it's passing the whole object so it brings the replacement type with it to be further passed to the redis setter. `ReplacementMessage.action_type` is the replacement type.